lib/util/mime_utils.js

```diff
@@ -16,14 +16,35 @@ goog.require('shaka.util.ManifestParserUtils');
 shaka.util.MimeUtils = class {
   /**
    * Takes a MIME type and optional codecs string and produces the full MIME
-   * type.
+   * type. Also remove the codecs for raw formats.
    *
    * @param {string} mimeType
    * @param {string=} codecs
    * @return {string}
    * @export
    */
   static getFullType(mimeType, codecs) {
+    let fullMimeType = mimeType;
+    if (codecs) {
+      if (!shaka.util.MimeUtils.RAW_FORMATS.includes(mimeType)) {
+        fullMimeType += '; codecs="' + codecs + '"';
+      } else {
+        fullMimeType += '; codecs=""';
```

Comment:
I've never seen `; codecs=""` before.  Wouldn't you just drop this code path and let `fullMimeType = mimeType`?

Reply:
Changed!